src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -1,8 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
TestAppDomainProcessExitEvenHandlerNotLeaking
```powershell
Assert.DoesNotContain() Failure
Found:    (filter expression)
In value: Delegate[] [EventHandler { Method = Void DisposeOnShutdown(System.Object, System.EventArgs), Target = null }, EventHandler { Method = Void CurrentDomain_ProcessExit(System.Object, System.EventArgs), Target = null }]
   at PSTests.Sequential.RunspaceTests.TestAppDomainProcessExitEvenHandlerNotLeaking() in D:\a\1\s\test\xUnit\csharp\test_Runspace.cs:line 127
```